### DISCUSSION TITLE
Preview Text color in the Color Scheme tab

### DISCUSSION THREAD

**Discussion by RoccoMarco** [jgraph/drawio#2747] [Q&A]
Hello everyone,
I am configuring draw.io to show me different groups of color schemes and this feature is amazing. The problem I am trying to solve is to get a preview of the color of the font for each preset.

For example if I preset
```
      {
        "fill": "#FF0000",
        "stroke": "#FF0000",
        "font": "#FFFFFF"
      },
```
I will see a RED rectange in my tab while I would like to see a Red rectangle with some white text inside.

this issue gets more annoying if I try to preset this
```
      {
        "fill": "none",
        "stroke": "none",
        "font": "#FF0000"
      },
```

as this will look like a totally transparent block. I am presetting such a block because the rectangle shape with text inside proves to be way more easier to be aligned in comparison to the text block. Any hint here would be highly appreciated

**Comment by davidjgraph** [comment]
I'm not sure what your question is.

**Reply by RoccoMarco** [reply]
Maybe a picture will help me clarify. The issue is there is no preview of the text style in the "Style" tab
![Untitled Diagram drawio](https://user-images.githubusercontent.com/5000233/162752901-7670d35a-5712-4c9f-b1a5-3b2623520b98.png)
